### PR TITLE
:construction: Add introduction

### PR DESCRIPTION
**Pull request by nymann**

### CODE REVIEW

File: project_report/tex/introduction.tex [hunk 1]
```diff
@@ -1 +1,21 @@
...
+well as a good idea of how such a project would progress. During the project,
+the group will learn how to create and set up an Open Platform Communications
+Unified Architecture, OPC-UA, client, and how to make it communicate directly
+to a physical machine. The group will learn to host a web server, from which the
+machine can be controlled, and access it as a website. The group will also have
+the chance to learn and use a new scripting language, JavaScript. The gained
+experience is the main motivation of this project.
+
+% Problem formulering
+
```
Comment (Kechr) [inline project_report/tex/introduction.tex]: Maybe add something like this here: 

```tex
\subsection{Problem Statement}
In the table \ref{table:problem-statement}, the finished problem, the problem statement and related questions are listed.
\begin{table}[ht]
    \begin{tabularx}{\textwidth}{|>{\RaggedRight}p{4cm}|>{\RaggedRight}X|}
        \hline
        \textbf{Problem} & The current production line is not effecient enough to keep up with the demand of the beer, while still maintaining a quality product\\
        \hline
        \textbf{Problem Statement} & How to control and optimise the brewing machine, to maximise the production of high quality beer\\
        \hline
        \textbf{Related questions} & 
            \begin{itemize}
                \item How can we optimise the production?
                \item How can we utilise calculus and linear algebra to provide a meaningful overview of the production line, based on statistics?
                \item How can we create a web based frontend for the MES?
                \item How can we separate the different aspects of the system (separation of concerns)
            \end{itemize}
        \\ 
        \hline
    \end{tabularx}
    \caption{Problem statement showcase} 
    \label{table:problem-statement}
\end{table} 
```

File: project_report/tex/introduction.tex [hunk 2]
```diff
@@ -1 +1,21 @@
 \section{Introduction}
+% Motivation from project proposal
+As students, this semester project gives the group great learning experience as
+well as a good idea of how such a project would progress. During the project,
+the group will learn how to create and set up an Open Platform Communications
+Unified Architecture, OPC-UA, client, and how to make it communicate directly
+to a physical machine. The group will learn to host a web server, from which the
+machine can be controlled, and access it as a website. The group will also have
+the chance to learn and use a new scripting language, JavaScript. The gained
+experience is the main motivation of this project.
```
Comment (Kechr) [inline project_report/tex/introduction.tex]: main motivation -> primary motivation (maybe?)

**Review by Kechr — CHANGES_REQUESTED** [review]
We might want to add section 4 Solutions and 5 Objectives to this introduction aswell. What do you think?

File: project_report/tex/introduction.tex (hunk 1 above)
Comment (nymann) [reply project_report/tex/introduction.tex]: :+1:

**Comment by nymann** [comment]
> We might want to add section 4 Solutions and 5 Objectives to this introduction aswell. What do you think?

Added

**Review by Kechr — APPROVED** [review]

File: project_report/tex/introduction.tex [hunk 3]
```diff
@@ -1 +1,89 @@
 \section{Introduction}
+
+% Motivation from project proposal
+As students, this semester project gives the group great learning experience as
+well as a good idea of how such a project would progress. During the project,
+the group will learn how to create and set up an Open Platform Communications
+Unified Architecture, OPC-UA, client, and how to make it communicate directly
+to a physical machine. The group will learn to host a web server, from which the
+machine can be controlled, and access it as a website. The group will also have
```
Comment (Joergenson) [inline project_report/tex/introduction.tex:9]: line: 9"access it" the machine or the webserver?

**Review by Joergenson — CHANGES_REQUESTED** [review]
Not much to say. Looks good my dude

**Review by Joergenson — APPROVED** [review]

**Pull request merged**